README.md

```diff
@@ -824,36 +824,31 @@ cf set-env <YOUR_APP> LOG_RATELIMIT '1000'
 
 For the third-party integrations explained above, in addition to the metrics collected by the agents, custom runtime metrics are provided via telegraf.
 This configuration also has a filtering mechanism that allows users to specify metrics they allow or deny for the vendor they are using.
-To filter the emission of custom runtime metrics, users should provide a list of prefixes of the metrics they want to allow/deny using the environment variables listed below.
+To filter the ingestion of custom runtime metrics to third party APMs, users should provide a list of prefixes of the metrics they want to allow/deny using the environment variables listed below.
 
 Note: Custom database metrics cannot be filtered by name, to turn them off, the `APPMETRICS_INCLUDE_DB` environment variable should be set to false. 
 
-#### APM_METRICS_FILTER_DENY
+#### APM_METRICS_FILTER_ALLOW
 
-List of prefixes for the metrics to be denied. 
+List of prefixes for the metrics to be allowed. By default, all metrics are allowed, even if they are not specified via this env var.
 
-For example, to deny all metrics starting with jetty or mx.runtime, the environment variable should be set to `jetty,mx.runtime`:
+For example, to deny all metrics except the session metrics, `APM_METRICS_FILTER_ALLOW` should be set to `mx.runtime.stats.sessions`:
```

Comment:
As we're in the `ALLOW` section, let's rephrase it to
```suggestion
For example, to allow only the session metrics, `APM_METRICS_FILTER_ALLOW` should be set to `mx.runtime.stats.sessions`:
```

Reply:
applied